Updated security policy (#8795)
keycloak/keycloak: SECURITY.md

# Security Policy

The Keycloak team takes security very seriously, and aim to resolve issues as quickly as possible. Building secure
software is a continuous process, and can always be improved. As such we welcome reports on potential security 
vulnerabilities, as well as suggestions around hardening the software and our process.

## Reporting a suspected vulnerability

> It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until

Reviewer: ```suggestion
We strongly encourage disclosing suspected vulnerabilities responsibly, and only disclose them publicly
```

Author: 'strongly encourage' is too soft - should be written more as a requirement, rather than a suggestion

Reviewer: I don't mind either formulation, though I incline a bit more to the softer variant. Both original formulations here and below look good to me, so feel free to merge it as is.

after they have been analysed and a fix is available.

> To report a security vulnerability send an email to keycloak-security@googlegroups.com.

Reviewer: ```suggestion
To report a security vulnerability, send an email to keycloak-security@googlegroups.com.
```

If you would like to work with us on a fix for the security vulnerability, please include your GitHub username
in the above email, and we will provide you access to a temporary private fork where we can collaborate on a fix 
without it being disclosed publicly.

> Do *not* open a public issue, send a pull request, or disclose any information about the suspected vulnerability publicly.

Reviewer: ```suggestion
We ask you to *not* open a public issue, send a pull request, or disclose any information about the suspected vulnerability publicly.
```
